toolset/setup/linux/languages/xsp.sh

```diff
@@ -7,18 +7,6 @@ RETCODE=$(fw_exists ${IROOT}/xsp.installed)
   source $IROOT/xsp.installed
   return 0; }
 
-# get
-git clone git://github.com/mono/xsp
-cd xsp
-git checkout e272a2c006211b6b03be2ef5bbb9e3f8fefd0768
-
-# build
-./autogen.sh --prefix=$MONO_HOME --disable-docs
-make
-sudo make install
-
-# cleanup
-cd ..
-rm -rf xsp
+sudo apt-get install -y mono-fastcgi-server
```

Comment:
This might not be 100% correct...

XSP is mono web server similar to ASP.NET
standalone XSP server is installed when you do a mono-complete installation

but XSP can run over Apache or Nginx
In that case you need those extract mono-fastcgi-server dependencies

so xsp.sh is poorly named, it should be named mono-fastcgi and only some of the C# frameworks depend on that.